.github/workflows/python-tox.yml

```diff
@@ -24,16 +24,15 @@ jobs:
       max-parallel: 4
       matrix:
         python-version:
-          - {version: 3.6, tox: 36}
           - {version: 3.7, tox: 37}
           - {version: 3.8, tox: 38}
           - {version: 3.9, tox: 39}
```

Comment:
**Suggestion**: Might be worth adding python 3.10

Reply:
Fixed in https://github.com/wagtail/wagtail-factories/pull/54/commits/f0b250f23f0ec68dc0264a744876bfb478583df1